PWGCF/MultiparticleCorrelations/MuPa-DataMembers.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// Remarks:
+// 0. Starting with C++11, it's possible to initialize data members at declaration, so I do it here
+// 1. Use //!<! for introducing a Doxygen comment interpreted as transient in both ROOT 5 and ROOT 6.
+
+// a) Base list to hold all output objects ("grandmother" of all lists);
+// *) QA;
+// *) Control event histograms;
+// *) Particle weights;
+
+// a) Base list to hold all output objects ("grandmother" of all lists):
+OutputObj<TList> fBaseList{"Task => ...", OutputObjHandlingPolicy::AnalysisObject, OutputObjSourceType::OutputObjSource};
```

Comment:
Yes, I am aware of `HystogramRegistry`, but currently, I do not use them, because:

1. it's easier to port my old code from `AliPhysics` by using the standard handling of histograms, which is still supported in O2; 
2. I couldn't find quickly an example of how to handle multidimensional arrays of histograms with `HystogramRegistry`

I will consider switching to `HystogramRegistry` later.

Reply:
Let's move forward with the current approach